### PR TITLE
:sparkles: improve audit log external origin events

### PR DESCRIPTION
**Pull request by superalex**

### CODE REVIEW

File: frontend/src/app/main/repo.cljs [hunk 1]
```diff
@@ -93,12 +93,12 @@
...
                     (str/starts-with? nid "get-") :get
                     :else :post)
-
         request   {:method method
                    :uri (u/join cf/public-uri "api/rpc/command/" nid)
                    :credentials "include"
                    :headers {"accept" "application/transit+json,text/event-stream,*/*"
-                             "x-external-session-id" (cf/external-session-id)}
+                             "x-external-session-id" (cf/external-session-id)
+                             "x-event-origin" (:app.main.data.events/origin (meta params))}
```
Comment (superalex) [inline frontend/src/app/main/repo.cljs]: Could we avoid using `:app.main.data.events/origin` ?

Comment (niwinz) [reply frontend/src/app/main/repo.cljs]: yes, requiring an alias:
```
[app.main.data.events :as-alias ev]
```
on requires

Comment (niwinz) [reply frontend/src/app/main/repo.cljs]: that will resultt on `::ev/origun`

File: backend/src/app/rpc/commands/teams.clj [hunk 2]
```diff
@@ -917,6 +918,14 @@
                    ::quotes/team-id (:id team)
                    ::quotes/incr (count emails)}))
 
+      (audit/submit! cfg
+                     {::audit/type "command"
```
Comment (niwinz) [inline backend/src/app/rpc/commands/teams.clj]: Should be type `action`

**Pull request merged**